purchase_line_product_image/models/purchase_order.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2017
+#   @Lucky Kurniawan <kurniawanluckyy@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    product_image = fields.Binary(
+        "image", related='product_id.product_tmpl_id.image_medium',
+        help="Small-sized image of the product. It is automatically "
```

Comment:
All the attributes but `related` are superfluous and can be removed. Using `related` will bring over all the attributes from the original declaration, so you only need to declare attributes that are being changed 😉

Reply:
hello @lasley, thanks for your review
can you help me give some example about "so you only need to declare attributes that are being changed"?
thanks

Reply:
@lasley remove related and use "@api.onchange"?

Reply:
yes, it's done